Javascript_Assignment_1.html

```diff
@@ -0,0 +1,61 @@
+<html>
+<head>
+
+	<script type="text/javascript">
+
+	function occurenceArray(){	
+		var sequencel,array_len,i,j;
+		var array=new Array();
```

Comment:
You can directly create an array instead of creating an instance of it and then assign a value like
var array =  sequence.split(",");